plugins/azure-devops/package.json

```diff
@@ -31,6 +31,7 @@
     "@backstage/core-components": "^0.7.2",
     "@backstage/core-plugin-api": "^0.1.12",
     "@backstage/errors": "^0.1.4",
+    "@backstage/plugin-azure-devops-backend": "^0.1.4",
```

Comment:
Hmm so - this is a little tricky here to add this dependency as we can't add backend plugins as dependencies to frontend packages for reasons.

Got a couple of choices here, pick which one you think is easiest or best for you.

1) Create a seperate package and depend on that in the frontend and backend plugins that houses the common types or enums `plugin-azure-devops-common` or something
2) Duplicate the enumns for now in the stories
3) Do a relative direct import and `eslint-ignore` that line

I'm leaning to either 2 or 3 is easiest for you, but either I think is fine.

Reply:
Created new `@backstage/plugin-azure-devops-common` package.